utils.ts

```diff
@@ -146,6 +146,13 @@ export async function setupRepo(options: RepoOptions) {
 	}
 }
 
+function splitInCliArgs(task: string) {
+	return task
+		.split(/("[^"]+"|[^\s"]+)/gim)
```

Comment:
this can break down if you wanted to pass a literal `\"`. Another way to support aguments for commands would be to require configs to pass an array of arrays eg
`test:[['test:ci','arg-1,'arg2 with spaces'],['test:e2e','another arg']]`  then we can take those arrays if the first one matches

Reply:
Implemented this here so you can check and merge what looks better:
- https://github.com/vitejs/vite-ecosystem-ci/pull/234

I'm actually fine with either approach. #234 isn't as clean as it seemed at the end though.